revelatormodsystem.py

```diff
@@ -2,49 +2,67 @@
 import sys
 import os
 
-scriptextract = False
+script_extract_folder = ''
 basepath = ''
+mod_folder = ''
+
+
 def bytearrtomessage(barr):
-	result = ""	
+	result = ""

```

Comment:
Trailing white space, set whatever you're using to edit code to highlight those for you.